File: app/src/main/java/com/wyang/study/ui/fragment_second/CustomViewFragment.java
package com.wyang.study.ui.fragment_second;

import com.wyang.study.R;
import com.wyang.study.ui.base.BaseFragment;
import com.wyang.study.ui.widget.ChineseFlag;

import butterknife.BindView;

/**
 * Created by weiyang on 2019-10-31.
 */
public class CustomViewFragment extends BaseFragment {
    @BindView(R.id.cf1)
    ChineseFlag cf1;
    @BindView(R.id.cf2)
    ChineseFlag cf2;

    @Override
    protected int getLayoutId() {
        return R.layout.fragment_custom_view;
    }

    @Override
    protected void initView() {
        cf1.setDrawLine(false);
        cf2.setDrawLine(true);
    }
}

File: app/src/main/java/com/wyang/study/ui/fragment/custom/CustomViewFragment.kt
package com.wyang.study.ui.fragment.custom

import com.w6539android.base.base.fragment.BaseFragment
import com.wyang.study.databinding.FragmentCustomViewBinding

class CustomViewFragment : BaseFragment<FragmentCustomViewBinding>() {

    override fun initialize() {
        mBinding.cf1.setDrawLine(false)
        mBinding.cf2.setDrawLine(true)
    }
}